src/chrome/content/rules/Freifunk_Gera-Greiz.xml

```diff
@@ -0,0 +1,17 @@
+<ruleset name="Freifunk Gera-Greiz">
+	<target host="freifunk-gera-greiz.de"/>
+	<target host="www.freifunk-gera-greiz.de"/>
+
+	<test url="http://www.freifunk-gera-greiz.de/favicon.ico"/>
+
+	<target host="gera.freifunk.net"/>
+	<target host="greiz.freifunk.net"/>
+	<target host="gera-greiz.freifunk.net"/>
```

Comment:
I used [Freifunk-Goettingen.de.xml](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Freifunk-Goettingen.de.xml) as an example for that rule.